bundles/org.openhab.binding.dominoswiss/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,127 @@
+			<parameter name="ipAddress" type="text" required="true">
+				<label>IP or Host Name (192.168.1.100, 1318) localhost 5500</label>
+				<description>The IP or host name of the Dominoswiss EGate Server</description>
+			</parameter>
+			<parameter name="port" type="integer" required="true" min="1" max="65335">
+				<label>Dominoswiss EGat Web Port</label>
+				<description>Port interface of the Dominoswiss EGate Server</description>
+				<default>1318</default>
+			</parameter>
+		</config-description>
+	</bridge-type>
+
+	<thing-type id="blind">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="egate"/>
+		</supported-bridge-type-refs>
+		<label>Blinds controlled by Dominoswiss</label>
+		<description>Provides various controlcommands for Dominoswiss receivers</description>
+		<channels>
+			<channel id="pulseUp" typeId="pulseUp"/>
+			<channel id="pulseDown" typeId="pulseDown"/>
+			<channel id="continuousUp" typeId="continuousUp"/>
+			<channel id="continuousDown" typeId="continuousDown"/>
+			<channel id="stop" typeId="stop"/>
+			<channel id="shutter" typeId="shutter"/>
+			<channel id="tilt" typeId="tilt"/>
+			<channel id="tiltUp" typeId="tiltUp"/>
+			<channel id="tiltDown" typeId="tiltDown"/>
+		</channels>
+		<properties>
+			<property name="vendor">Dominoswiss</property>
+			<property name="modelId"></property>
+			<property name="name"></property>
+			<property name="id"></property>
+		</properties>
+		<config-description>
+			<parameter name="id" type="text" required="true">
+				<label>ID Address</label>
+				<description>Blinds are identified by their ID address</description>
+			</parameter>
+		</config-description>
+	</thing-type>
+
+	<channel-type id="pulseUp">
+		<item-type>Number</item-type>
+		<label>Pulse Up</label>
+		<description>Send one pulse up</description>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="pulseDown">
+		<item-type>Number</item-type>
+		<label>Pulse Down</label>
+		<description>Send one pulse down</description>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="continuousUp">
+		<item-type>Number</item-type>
+		<label>Continuous Up</label>
+		<description>Send continuous up command to blind</description>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="continuousDown">
+		<item-type>Number</item-type>
+		<label>Continous Down</label>
+		<description>Send continuous down command to blind</description>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="stop">
+		<item-type>Number</item-type>
+		<label>Stop</label>
+		<description>Send stop impulse to stop the blinds</description>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="shutter">
+		<item-type>UpDownType</item-type>
+		<label>shutter</label>
+		<description>Handle the commands up/down/stop </description>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="tilt">
+		<item-type>UpDownType</item-type>
+		<label>shutter</label>
```

Comment:
This label is a double up of the shutter channel. When you use the 'add thing as equipment' feature, openHAB  will auto create items using the label of each channel in the name so this not being unique may/will create an issue. Can this label be changed to something more descriptive that makes the channels exact use more clear?